aten/src/ATen/native/native_functions.yaml

```diff
@@ -5737,6 +5737,20 @@
     CPU: foreach_tensor_add_scalar_kernel_slow_
     CUDA: foreach_tensor_add_scalar_kernel_cuda_
 
+- func: _foreach_add.List(Tensor[] tensors1, Tensor[] tensors2) -> Tensor[]
+  device_guard: False
+  variants: function
+  dispatch:
+    CPU: foreach_tensor_add_list_kernel_slow
+    CUDA: foreach_tensor_add_list_kernel_cuda
+
+- func: _foreach_add_.List(Tensor[](a!) self, Tensor[] other) -> ()
```

Comment:
Shouldn't this be `Tensor(a!)[]` ? I'm reading `Tensor[](a!)` as "the list self is being modified in-place", and "Tensor(a!)[]" as "the list self is not being modified, but its constituent tensors are being modified in-place"